src/theme/class-theme.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * The Theme specific functionality.
+ *
+ * @since   1.0.0
+ * @package Inf_Theme\Theme
+ */
+
+namespace Inf_Theme\Theme;
+
+use Eightshift_Libs\Core\Service;
+use Inf_Theme\Helpers\General_Helper;
+
+/**
+ * Class Theme
+ */
+class Theme implements Service {
+
+  /**
+   * Register all the hooks
+   *
+   * @return void
+   *
+   * @since 1.0.0
+   */
+  public function register() : void {
+    \add_action( 'wp_enqueue_scripts', [ $this, 'enqueue_styles' ] );
+    \add_action( 'wp_enqueue_scripts', [ $this, 'enqueue_scripts' ] );
+
+    /**
+     * Optimizations
+     *
+     * This will remove some default functionality, but it mostly removes unnecessary
+     * meta tags from the head section.
+     */
+    \remove_action( 'wp_head', 'print_emoji_detection_script', 7 );
```

Comment:
I'll leave this, but same as with svg, leave a notice that such a theme wouldn't pass wp.org review because there is a rule that forbids removing core actions.

Reply:
👍